src/ONYXKEYS.js

```diff
@@ -89,11 +89,8 @@ export default {
     // Contains the users's block expiration (if they have one)
     NVP_BLOCKED_FROM_CONCIERGE: 'private_blockedFromConcierge',
 
-    // SDK token used to communicate with Plaid API
-    PLAID_LINK_TOKEN: 'plaidLinkToken',
-
-    // List of bank accounts returned by Plaid
-    PLAID_BANK_ACCOUNTS: 'plaidBankAccounts',
+    // Plaid data (tokens, bank accounts ...)
+    PLAID_DATA: 'plaidData',
```

Comment:
Ah yes ... it felt better to me to gather them all in the same object (linkToken, accessToken and bankAccounts), as they all come from Plaid, from the same flow, and it's easier to then wipe the object out rather than separately when needed.

PLAID_DATA still makes sense to gather plaidBankAccounts and accessToken, as they come together.
But I agree maybe I should have let PLAID_LINK_TOKEN separately as it's a separate user action.

I'd need to create a PHP PR for it though before we can change it here and merge.

Reply:
https://github.com/Expensify/Web-Expensify/pull/34249